### PR TITLE
ENH add weak lensing deblending

### PR DESCRIPTION
**Pull request by beckermr**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge/staged-recipes This one is ready for review!

File: recipes/lsstdesc_weaklensingdeblending/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/LSSTDESC/WeakLensingDeblending/archive/v{{ version }}.tar.gz
+  sha256: 2741773f3274ec6216667bc7d2e0e08c9e3ec279ef1a7c8bb6bffec8323ba68a
+
+build:
+  number: 0
+  skip: True  # [win]
```
Comment (xhochy) [inline recipes/lsstdesc_weaklensingdeblending/meta.yaml]: What's the reason for this not being `noarch: python`?  Missing dependencies? Then that's no problem and we can ignore the failing Windows CI on merge.

Comment (beckermr) [reply recipes/lsstdesc_weaklensingdeblending/meta.yaml]: Woops you are right!

Comment (beckermr) [reply recipes/lsstdesc_weaklensingdeblending/meta.yaml]: done!

**Comment by beckermr** [comment]
Ready for another look @xhochy @conda-forge/staged-recipes!

**Comment by beckermr** [comment]
There are missing deps on windows which explains those failures.

**Comment by beckermr** [comment]
Thank you!

**Pull request merged**